indica/operators/centrifugal_asymmetry.py

```diff
@@ -135,20 +149,21 @@ def __call__(  # type: ignore
 
         # mypy on the github CI suggests that * is an Unsupported operand type
         # between float and DataArray, don't know how to fix yet so for now ignored
-        toroidal_rotation = 2.0 * ion_temperature * asymmetry_parameter  # type: ignore
-        toroidal_rotation /= impurity_mass * (
-            1.0
-            - (mean_charge * main_ion_mass * Zeff * electron_temp)  # type: ignore
-            / (impurity_mass * (ion_temperature + Zeff * electron_temp))
+        return (
```

Comment:
The change to using `np.sqrt` and `spc.speed_of_light` is good, but it might be better for readability to have the calculation done in stages on a single variable (`toroidal_rotation`) as before. Is there any specific reason to do the calculation in the `return` statement?

Reply:
I found it easier to compare to equation 2.12 written like this. I also didn't like that we kept in-place modifying the `toroidal_rotation` variable which only had the correct value when it was returned.